lib/killbill/gen/api/pagination.rb

```diff
@@ -41,7 +41,7 @@ def initialize()
 
         def to_java()
           # conversion for iterator [type = java.util.Iterator]
-          @iterator = Killbill::Plugin::Model::EnumeratorIterator.new(@iterator)
```

Comment:
> Hi @pierre , any updates?

Sorry for the slow reply, but I was :airplane:  in the past two weeks.

>  Please inform me if this change is not necessary, I will update my pr. Thanks.

In general, you should never do any manual change in the `gen` package as the code is auto-generated by the [killbill-java-parser](https://github.com/killbill/killbill-java-parser).

But regardless, this change seems incorrect. `EnumeratorIterator` is a class to convert a Ruby `Enumerable` or `Enumerator` to a `java.util.Iterator`. See [here](https://github.com/killbill/killbill-java-parser/blob/1967e802bbb2bbf2ae4a193832e90d0941c46ad3/src/main/java/com/ning/killbill/generators/ruby/JRubyPluginGenerator.java#L659). It should only be used in the `to_java` API conversions behind the scenes (the core of the `killbill` gem is pure Ruby, not JRuby).